src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -347,19 +347,15 @@
   <div class="status-block">
     <h2 class="heading-small">{{ t('Mediation was unsuccessful') }}</h2>
     <p>{{ t('You weren’t able to resolve your claim against {{ defendantName }} using mediation.', { defendantName: claim.claimData.defendant.name }) }}</p>
-    {% if claim.pilotCourt === YesNoOption.YES %}
-      <p>{{ t('The court will review the case. We’ll contact you to tell you what to do next.') }}</p>
-    {% else %}
-      <p>{{ t('You’ll have to go to a hearing. We’ll contact you with the details.') }}</p>
-    {% endif %}
+    <p>{{ t('The court will review the case. We’ll contact you to tell you what to do next.') }}</p>
```

Comment:
if this hasn't failed a test then it means there are tests missing, can you please add them?

Reply:
there were tests failing which are now updated.